enhancements/ingress/haproxy-reload-interval.md

```diff
@@ -0,0 +1,163 @@
+---
+title: haproxy-reload-interval
+authors:
+  - "@Ethany-RH"
+reviewers:
+  - "?"
+approvers:
+  - "?"
+api-approvers: # necessary?
+  - "?"
+creation-date: 2022-07-01
+last-updated: 2022-07-01
+tracking-link:
+  - "https://issues.redhat.com/browse/NE-586"
+see-also:
+replaces:
+superseded-by:
+---
+
+# Reload Interval in HAProxy
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`.
+- [ ] Design details are appropriately documented from clear requirements.
+- [ ] Test plan is defined.
+- [ ] graduation criteria for dev preview, tech preview, GA
+- [ ] User-Facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/).
+
+## Summary
+
+Add an API field to configure OpenShift router's `RELOAD_INTERVAL` environment variable so that administrators can define the minimum frequency the router is allowed to reload to accept new changes.
```

Comment:
```suggestion
Add an API field to configure OpenShift router's `RELOAD_INTERVAL` environment variable so that administrators can define the minimum frequency the router is allowed to reload in order to publish new changes.
```